xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/AudioDecoder.h

```diff
@@ -63,167 +69,167 @@ namespace kodi
 namespace addon
 {
 
-  class CInstanceAudioDecoder : public IAddonInstance
+class CInstanceAudioDecoder : public IAddonInstance
+{
+public:
+  //==========================================================================
+  /// @brief Class constructor
+  ///
+  /// @param[in] instance             The from Kodi given instance given be
```

Comment:
I cannot really decode this text :)

Reply:
There I not changed :grinning:, only added by the guidelines change to another position :smirk:

Reply:
Is changed to
```cpp
  /// @param[in] instance The addon instance class handler given by Kodi
  ///                     at \ref kodi::addon::CAddonBase::CreateInstance(...)
```
and links to them on webview,